frontend/app/services/lookup-service.server.ts

```diff
@@ -125,7 +125,16 @@ const applicationTypesSchema = z.object({
   nameEn: z.string(),
   nameFr: z.string(),
 });
-
+const firstNationsTypeSchema = z.object({
+  id: z.string(),
+  nameEn: z.string().optional(),
+  nameFr: z.string().optional(),
+});
+const getAllFirstNationsEnumTypeSchema = z.object({
+  id: z.string(),
+  nameEn: z.string().optional(),
+  nameFr: z.string().optional(),
+});
```

Comment:
These validators will allow empty strings and all-whitespace strings. Was this intentional?

Reply:
i followed what was done on other pages, we will fix in a follow up PR, for now we just want to get the question to the page and available for use.